resources/metadata/abapEnvironmentPullGitRepo.yaml

```diff
@@ -34,9 +34,9 @@ spec:
         - STEPS
         mandatory: true
         secret: true
-      - name: repositoryName
-        type: string
-        description: Specifies the name of the Repository (Software Component) on the SAP Cloud Platform ABAP Environment system
+      - name: repositoryNames
+        type: '[]string'
```

Comment:
Are upper quotes viable for type declaration in YAML? I think you can simply write `type: []string`

Reply:
the quotes are necessary